### DISCUSSION TITLE
Down arrow key doesn't work with bindkey -v

### DISCUSSION THREAD

**Discussion by resting** [marlonrichert/zsh-autocomplete#720] [Q&A]
The down arrow key doesn't work whenever I enable vim mode with `bindkey -v`
It works if it's not enabled.
Is there a workaround with vim mode?
Am not able to enter the menu selection currently.

Tried adding the following to `.profile` but it didn't change anything:
```
bindkey -M vicmd "k" up-line-or-beginning-search
bindkey -M vicmd "j" down-line-or-beginning-search
```

OS: macOS

**Comment by marlonrichert** [comment]
[As mentioned in the Readme](https://github.com/marlonrichert/zsh-autocomplete/blob/main/README.md#keyboard-shortcuts), Autocomplete adds these keybindings only to the `main` keymap. So, if you want them to end up in `viins`, run `bindkey -v` _before_ you source Autocomplete.
